### PR TITLE
Feature EPL-392: React JS architecture

### PR DESCRIPTION
**Pull request by sebastianbarrozo**

### CODE REVIEW

File: settings.gradle [hunk 1]
```diff
@@ -2,14 +2,11 @@ pluginManagement {
...
         gradlePluginPortal()
-        maven {
-            url 'https://repo.futit.cloud/repository/maven-public-snapshots'
-        }
         maven {
             url 'https://repo.futit.cloud/repository/maven-public-releases'
         }
         maven {
-            url 'https://repository-cdn.liferay.com/nexus/content/repositories/liferay-public-releases'
+            url 'https://repo.futit.cloud/repository/maven-public-snapshots'
```
Comment (juabaez) [inline settings.gradle:9]: why snapshots ?

**Review by juabaez — CHANGES_REQUESTED** [review]

File: settings.gradle (hunk 1 above)
Comment (sebastianbarrozo) [reply settings.gradle:9]: We still don't have a stable release for EtendoRX's jar version

**Review by juabaez — APPROVED** [review]

File: .gitignore [hunk 2]
```diff
@@ -1,13 +1,7 @@
...
-!gradle/**
-!gradle.properties
-!settings.gradle
-!.run
-!.run/**
-!src-rx
+.DS_Store
+/bin
+/src-rx/logs
+/modules_rx
```
Comment (isaiasb-etendo) [inline .gitignore:4]: Should the /modules folder also be ignored?

Comment (sebastianbarrozo) [reply .gitignore:4]: It is already included below in the same file

**Review by RubenEtendo — APPROVED** [review]

**Review by valeg-etendo — APPROVED** [review]

**Pull request merged**